janus_core/calculations/descriptors.py

```diff
@@ -78,39 +123,59 @@ def __init__(
         write_kwargs : Optional[ASEWriteArgs],
             Keyword arguments to pass to ase.io.write if saving structure with
             results of calculations. Default is {}.
-        log_kwargs : Optional[dict[str, Any]]
-            Keyword arguments to pass to `config_logger`. Default is {}.
-        tracker_kwargs : Optional[dict[str, Any]]
-                Keyword arguments to pass to `config_tracker`. Default is {}.
         """
-        (write_kwargs, log_kwargs, tracker_kwargs) = none_to_dict(
-            (write_kwargs, log_kwargs, tracker_kwargs)
+        (
+            read_kwargs,
+            calc_kwargs,
+            log_kwargs,
+            tracker_kwargs,
+            write_kwargs,
+        ) = none_to_dict(
+            (
+                read_kwargs,
+                calc_kwargs,
+                log_kwargs,
+                tracker_kwargs,
+                write_kwargs,
+            )
         )
```

Comment:
Duplicate?

Reply:
read_kwargs stays as a duplicate now, so I can set the default index for each calculation, but otherwise these should now only by the kwargs not set within the base calc